lib/libc/netdb/lib_getnameinfo.c

```diff
@@ -23,6 +23,7 @@
 #include <tinyara/config.h>
 
 #include <netdb.h>
+#include <arpa/inet.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?